### PR TITLE
upi/vsphere: Make cidr_to_ip.sh more verbose

### PR DESCRIPTION
**Pull request by LorbusChris**
/hold

### CODE REVIEW

File: upi/vsphere/ipam/cidr_to_ip.sh [hunk 1]
```diff
@@ -33,22 +32,24 @@ is_ip_address() {
...
 
 function produce_output() {
-  if [[ "${network}" == "null" ]]
-  then
+  if [[ "${network}" == "null" ]]; then
     jq -n \
       --arg ip_address "$(get_reservation)" \
       '{"ip_address":$ip_address}'
-	exit 0
+	  exit 0
```
Comment (patrickdillon) [inline upi/vsphere/ipam/cidr_to_ip.sh:52]: why the extra space? seems like it should be dedented, not indented?

Comment (LorbusChris) [reply upi/vsphere/ipam/cidr_to_ip.sh]: thanks, my editor had wrong number of spaced per tab configured here. fixed!

**Comment by jcpowermac** [comment]
Cleaned up IPAM. Let's make sure first that UPI completes install then I will approve

**Comment by jcpowermac** [comment]
@LorbusChris terraform and the script worked as expected something else is wrong with master.
/lgtm

**Pull request merged**